test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -325,18 +336,13 @@ private void wipeCluster() throws Exception {
             wipeClusterSettings();
         }
 
-        if (hasXPack && false == preserveRollupJobsUponCompletion()) {
-            try {
-                wipeRollupJobs();
-            } catch (ResponseException e) {
-                // Temporary work around for mixed clusters, some of which don't have rollup
-                if (    // The request was made to node without rollup
-                        e.getResponse().getStatusLine().getStatusCode() != 400
-                        // or the request was made to a node with rollup and it sent something to a node without it
-                        && false == e.getMessage().contains("No handler for action [cluster:monitor/xpack/rollup/get]")) {
-                    throw e;
-                }
-            }
+        /*
+         * Rollups were introduced in 6.3.0 so any cluster that contains older
+         * nodes won't be able to do *anything* with rollups, including cleanup.
+         */
+        if (hasXPack && nodeVersions.first().onOrAfter(Version.V_6_3_0)
```

Comment:
should there be a comment here explaining why it is sufficient that any node has xpack, and not necessary to have all nodes have xpack?

I guess this leaks some things about our test infrastructure that may invalidate such comments in the future, so let me know if you think that would just be more confusing!

Reply:
So, yeah. I think I'd like to do a follow up to break apart "any have xpack" and "all have xpack" because I think the `skip: features: x-pack` in yml tests really want *all* nodes to have x-pack. And the rollup code here wants all nodes to have x-pack. But the template preservation code wants *any* nodes have x-pack.

Reply:
cool. this is sufficient for ILM for now, so that makes sense